045_amstrongnumberinarange.js

```diff
@@ -0,0 +1,19 @@
+let array = [];
+const amstrongRange = (arr) => {
+  let max = Math.max(arr[0], arr[1]);
+  let min = Math.min(arr[0], arr[1]);
+  for (let i = min; i <= max; i++) {
+    let sum = 0;
+    let num = i;
+    while (num > 0) {

```

Comment:
Yous should have written the functton to calculate the Armstrong as a separate function